libraries/docker_container.rb

```diff
@@ -423,11 +423,11 @@ def to_snake_case(name)
       ro_rootfs container.info['HostConfig']['ReadonlyRootfs']
       if container.info['NetworkSettings'] &&
          container.info['NetworkSettings']['Networks'] &&
-         container.info['NetworkSettings']['Networks'][new_resource.network_mode] &&
```

Comment:
Isn't this going to result in a unfound method error?

Reply:
@tas50 - it shouldn't cause that error.
It actually used to be this way and was broken as part of this commit -
https://github.com/chef-cookbooks/docker/commit/b4bf74025af6ddd77e129e479dd19556e552d203

So - in effect - this would revert back to the older behavior, which oddly does work.  But - now that you mention it and I looked at it a bit deeper there is a better way of handling this.  
I'm going to revise this PR to what I think the original intention is when it was originally added in
https://github.com/chef-cookbooks/docker/commit/f4f477e61cb6510e4ceac2f03a5bd0d37c3c3d68

There is a better way to handle this - and the way of handling this should have tests.
I'll post a much longer explanation of the problem and a solution with tests shortly.